### PR TITLE
TheDescent/Esd109

### PR DESCRIPTION
**Pull request by Ethan-Dav109**
Intro to Github Assignment

### CODE REVIEW

**Review by bjones1 — CHANGES_REQUESTED** [review]

File: TheDescent/esd109/The DescentESD109_updated.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# # Ethan Davis ESD109 The Descent CodeChat Editor
```
Comment (bjones1) [inline TheDescent/esd109/The DescentESD109_updated.py:1]: What is this? I want just one code file with the solution, not two.

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by Ethan-Dav109** [comment]
Updated it, I do not know what in the world those random numbers and symbols are, they look like they are being added in while it transfers. I removed the second file and updated the first one.
________________________________

File: TheDescent/esd109/The DescentESD109.py [hunk 2]
```diff
@@ -1,34 +1,44 @@
...
+# the game where you are given inputs, (the heights of the mountains), and where
+# you have to print an output, (the index of the mountain to fire on). The
+# inputs you are given are automatically updated according to your last actions.
+# There 5 test cases that are needed to be passed for this particular problem.
+# In order to pass each of these, a culmination of different types of loops and
+# if statements may be needed. Below is a simple diagram that I drew by hand
+# when I first began to solve this problem.\
+# \
+#
+# <span style="background-color: #2dc26b;">esd109:</span> Thank you for this
```
[the remark this answers is not in the document]
Comment (bjones1) [reply TheDescent/esd109/The DescentESD109.py]: I still see these in the file...

File: TheDescent/esd109/The DescentESD109_updated.py (hunk 1 above)
Comment (bjones1) [reply TheDescent/esd109/The DescentESD109_updated.py]: This looks like you resolved it -- click the "Resolve conversation" below so you can keep track of what's done and what still needs doing.

**Pull request merged**